.cirrus.yml

```diff
@@ -121,22 +124,22 @@ gating_task:
 
     gate_script:
         # N/B: entrypoint.sh resets $GOSRC (same as make clean)
-        - '/usr/local/bin/entrypoint.sh validate'
-        - '/usr/local/bin/entrypoint.sh lint'
-        - '${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/test/test_dot_cirrus_yaml.py'
+        - '/usr/local/bin/entrypoint.sh validate |& ${TS}'
```

Comment:
Did you test all (any) of these on failure? Pipes obscure exit status unless `-o pipefail` is set. (I see it mentioned in four places in the repo, but can't tell if any of them actually take effect in this particular CI setup).

Reply:
FWIW, I love the variable name!

Reply:
Yes, I'm concerned about the `pipefail` problem also.  I'll instrument the PR temporarily with a script that exits non-zero, and another script that doesn't exist, to see how both are handled.